.eslintrc.js

```diff
@@ -1,14 +1,30 @@
 module.exports = {
   root: true,
+  parserOptions: {
+    "parser": "babel-eslint",
+    "ecmaVersion": 8,
+    "sourceType": "module"
+  },
   env: {
-    node: true,
+    "browser": true,
+    "node": true
+  },
+  globals: {
+    "jsdom": true,
+    "Promise": true
   },
-  extends: ["plugin:vue/recommended" /*,"@vue/airbnb", "@vue/prettier"*/],
+  plugins: [
+    "jest",
+    "vue"
+  ],
+  extends: [
+    "eslint:recommended",
+    "plugin:vue/recommended",
+    "plugin:jest/recommended",
+    "plugin:vue-a11y/recommended"
```

Comment:
I don't actually like all of these default linting rules, I actually have a set of my own custom ones that I apply to my projects:

* https://github.com/tjw-lint/eslint-config-tjw-base
* https://github.com/tjw-lint/eslint-config-tjw-vue

But, while I think these defaults can be improved they're pretty acceptable and fairly unopinionated.

Vuetensils being unopinionated is one of the things I like about it (no typescript, no prettier, no yarn, etc). So I'm trying to stick with that approach here.